### PR TITLE
Support mobile devices

### PR DESCRIPTION
**Pull request by yujinakayama**
There're still some skewed layouts on mobile devices but it's better than the current desktop-only support.

## Before

rspec-before

## After

rspec-after

### CODE REVIEW

File: source/layouts/layout.slim [hunk 1]
```diff
@@ -15,11 +15,11 @@ html.no-js lang="en"
 
   head
     meta charset="UTF-8"
+    meta name="viewport" content="width=device-width, initial-scale=1"
     == yield_content(:head)
 
     title= current_page.data.title
 
-    == stylesheet_link_tag 'application', 'normalize'
```
Comment (JonRowe) [inline source/layouts/layout.slim]: Are we sure we import application in every page partial? I'm not even sure we have a page partial for every page :/

**Comment by myronmarston** [comment]
Any reason not to merge this?  I trust @yujinakayama :).

**Comment by JonRowe** [comment]
My concern has yet to be addressed, if @yujinakayama says we do in fact reimport it everywhere I'm okay with it, otherwise I'd like that addressed before merge.

**Comment by yujinakayama** [comment]
I changed the stylesheet handling by removing preprosessor concatenation and always importing `application.css` by browsers.

File: source/layouts/layout.slim (hunk 1 above)
Comment (yujinakayama) [reply source/layouts/layout.slim]: I've confirmed that all pages are properly displayed with this. Actually I want to refactor all the stylesheets but doing so is out of scope of this PR.

Comment (JonRowe) [reply source/layouts/layout.slim]: @yujinakayama ok I trust you, merge and deploy at your lesuire

**Pull request merged**